api/error.py

```diff
@@ -17,6 +17,14 @@ class ErrorCode(Enum):
         code=400,
         title='File exists',
         detail='Avatar file already exists')
+    AVATAR_IN_USE = dict(
+        code=400,
+        title='Avatar in use',
+        detail='Avatar cannot be deleted because it is still in use')
+    AVATAR_INTEGRITY_ERROR = dict(
+        code=400,
```

Comment:
Yes, all the error codes are a continuous numbering starting at 100, not HTTP status codes. And neither the error title nor the detail message isn't useful.